components/aws/sagemaker/tests/integration_tests/utils/__init__.py

```diff
@@ -99,6 +101,71 @@ def replace_placeholders(input_filename, output_filename):
     return output_filename
 
 
+def get_rule_image_registry(region, rule_type):
+    region_to_account = {}
+    if rule_type == "built-in":
+        region_to_account = {
```

Comment:
Oh boy this is the best we can do? Does the [SageMaker SDK](https://sagemaker.readthedocs.io/en/stable/) have a method that supports fetching these programatically?